go-tour/eqvbintrees.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"golang.org/x/tour/tree"
+)
+
+// Walk walks the tree t sending all values
+// from the tree to the channel ch.
+func Walk(t *tree.Tree, ch chan int) {
+	if t != nil {
```

Comment:
in golang usually use another approach: returns or small logic by condition and other logic is placed after that for improving readability